--- a/ImNodesEz.cpp
+++ b/ImNodesEz.cpp
@@ -179,6 +179,7 @@
     ImGui::TextUnformatted(title);

     ImGui::SetCursorScreenPos(input_pos);
+    ImGui::Dummy(ImVec2(0, 0));

     return result;
 }
@@ -243,6 +244,7 @@
     if (!ImNodes::IsOutputSlotKind(kind))
         item_offset_x = -item_offset_x;
     ImGui::SetCursorScreenPos(pos + ImVec2{item_offset_x, 0 });
+    ImGui::Dummy(ImVec2(0, 0));

     pos.y += ImMax(title_size.y, 2*CIRCLE_RADIUS) + g.Style.ItemSpacing.y;

@@ -326,6 +328,7 @@

     // Move cursor to the next column
     ImGui::SetCursorScreenPos(ImVec2{storage->GetFloat(ImGui::GetID("content-x")), storage->GetFloat(ImGui::GetID("body-y"))});
+    ImGui::Dummy(ImVec2(0, 0));

     PopStyleVar(2);

@@ -352,6 +355,7 @@

     // Set cursor screen position as it is recorded as the starting point in BeginGroup() for the item rect size.
     ImGui::SetCursorScreenPos(pos);
+    ImGui::Dummy(ImVec2(0, 0));

     // Render output slots in the next column
     if (snum) {
